freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -65,12 +70,90 @@ def handle(self, event):
         self.log_info(
             "Orignial nvrs of build in the advisory #{0} are: {1}".format(
                 db_event.search_key, " ".join(original_nvrs)))
+        # Get images by nvrs and then get their digests
+        original_images_digests = set(self.get_digests_by_nvrs(original_nvrs))
+        if not original_images_digests:
+            log.warning("The are no digests for NVRs: %s",
+                        ",".join(original_nvrs))
+            return []
+
+        index_images = self._pyxis.get_operator_indices()
+        # get latest bundle images per channel per index image filtered by higher version
+        all_bundles = self._pyxis.get_latest_bundles(index_images)
 
-        msg = "Skipping due to being blocked on further implementation for now."
+        bundles = self._pyxis.filter_bundles_by_digests(original_images_digests,
+                                                        all_bundles)
+        bundle_digests = []
+        for bundle in bundles:
+            if not bundle.get('bundle_path_digest'):
+                log.warning("Bundle %s doesn't have 'bundle_path_digests' set",
+                            bundle['bundle_path'])
+                continue
+            bundle_digests.append(bundle['bundle_path_digest'])
+        bundle_images = self._pyxis.get_images_by_digests(bundle_digests)
+
+        # now we have nvrs of bundles to rebuild
+        bundles_nvrs = self._filter_bundle_images_by_related_images(  # noqa: F841
+            bundle_images, original_images_digests)
 
         # Skip that event because we can't proceed with processing it.
         # TODO
-        # Next step would be queries Pyxis and getting the digests of these nvrs
+        # Now when we have bundle images' nvrs we can procceed with rebuilding it
+        msg = f"Skipping the rebuild of {len(bundles_nvrs)} bundle images " \
+              "due to being blocked on further implementation for now."
         db_event.transition(EventState.SKIPPED, msg)
         db.session.commit()
         return []
+
+    def _filter_bundle_images_by_related_images(self, bundle_images, digests):
+        """
+        Filter out bundle images that don't reference the original digests
+        of the operator/operand images
+
+        :param dict bundle_images: Mapping of bundle digest to list of bundle's
+        ContainerImages
```

Comment:
This is invalid RST. You need to indent these continuation lines. Could you please fix this in all of your new docstrings?
```suggestion
            ContainerImages
```